docs/conf.py

```diff
@@ -107,6 +107,7 @@ def gomod_versions(modules):
 
 mod_versions = gomod_versions(['github.com/intel/goresctrl'])
 myst_substitutions = {
+    'version': version,
     'golang_version': mod_versions['golang'],
     'goresctrl_version': mod_versions['github.com/intel/goresctrl']
 }
```

Comment:
Does this work @marquiz ?

Reply:
Argh, messed up with some testing 🤯 Thanks for noting, removed